pkgs/cli/src/api_server.zig

```diff
@@ -2,6 +2,25 @@ const std = @import("std");
 const api = @import("@zeam/api");
 const constants = @import("constants.zig");
 const event_broadcaster = api.event_broadcaster;
+const node = @import("@zeam/node");
+
+// Global chain reference for API access
+var global_chain: ?*node.BeamChain = null;
```

Comment:
i dont like global chain, you should register the routes on the api server using the forkchoice instance, not even chain